src/widgets/image/lv_image.c

```diff
@@ -749,6 +749,11 @@ static void draw_image(lv_event_t * e)
                         obj->coords.y1,
                         obj->coords.x1 + img->w - 1,
                         obj->coords.y1 + img->h - 1);
+
+            if(lv_obj_get_style_clip_corner(obj, LV_PART_MAIN)) {
```

Comment:
I think it should be independent of `clip_corner`.

- If only `clip_radius` is only the image will have radius
- If `clip_corner` is set too, the the children will be clipped as well. It uses a lot of memory for large radiuses, so I would leave to the user to control it.

Reply:
Do you mean to add a separate API to image: `lv_image_set/get_clip_radius`?

Reply:
I'm sorry I missed this comment. I meant just removing the `if(lv_obj_get_style_clip_corner(obj, LV_PART_MAIN)) {` condition so the radius is always applied if set.

Reply:
Done